frontend/fingertips-frontend/components/views/OneIndicatorOneAreaView/index.tsx

```diff
@@ -1,33 +1,28 @@
-import { OneIndicatorOneAreaDashboard } from '@/components/dashboards/OneIndicatorOneAreaDashboard';
+import { OneIndicatorOneAreaViewPlots } from '@/components/viewPlots/OneIndicatorOneAreaViewPlots';
 import { areaCodeForEngland } from '@/lib/chartHelpers/constants';
-import {
-  SearchParams,
-  SearchStateManager,
-  SearchStateParams,
-} from '@/lib/searchStateManager';
-
-type OneIndicatorOneAreaViewProps = {
-  searchState: SearchStateParams;
-};
+import { SearchParams, SearchStateManager } from '@/lib/searchStateManager';
+import { ViewProps } from '../ViewsContext';
 
 export default async function OneIndicatorOneAreaView({
   searchState,
-}: Readonly<OneIndicatorOneAreaViewProps>) {
+}: Readonly<ViewProps>) {
   const stateManager = SearchStateManager.initialise(searchState);
   const {
-    [SearchParams.AreasSelected]: areaCodes,
+    [SearchParams.AreasSelected]: areasSelected,
     [SearchParams.GroupSelected]: selectedGroupCode,
   } = stateManager.getSearchState();
 
-  const areasSelected = areaCodes ?? [];
+  if (areasSelected?.length !== 1) {
+    throw new Error('Invalid parameters provided to view');
+  }
 
-  const areaCodesToRequest =
-    selectedGroupCode && selectedGroupCode != areaCodeForEngland
-      ? [...areasSelected, areaCodeForEngland, selectedGroupCode]
-      : [...areasSelected, areaCodeForEngland];
+  const areaCodesToRequest = [areasSelected, areaCodeForEngland];
```

Comment:
`areasSelected` is an array here, so I think you may want to do:
```suggestion
  const areaCodesToRequest = [...areasSelected, areaCodeForEngland];
```